api/pom.xml

```diff
@@ -19,6 +19,12 @@
 			<type>jar</type>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-params</artifactId>
+			<version>5.7.0</version>
+			<scope>test</scope>
+		</dependency>
```

Comment:
Do we need this change?

Reply:
i introduced the change since i wrote junit 5 based tests @ibacher

Reply:
Ok, but you seem to just be using `assertTrue` and `assertFalse` from Jupiter (JUnit 5) rather than anything JUnit 5-specific (and, in fact, `@Before` is JUnit 4 rather than JUnit 5), so do we really need to add this in?

Reply:
hi @ibacher  so i dropped @before from jnit 4